webgpu/lessons/webgpu-inter-stage-variables.md

```diff
@@ -308,18 +308,20 @@ And we'd have to update our pipeline creation to use these
 
 ```
 
-And this would also work
+These modifications render the same:
 
 {{{example url="../webgpu-fragment-shader-builtin-position-separate-modules.html"}}}
 
 The point is, the fact that both shaders are in the same string in most WebGPU
 examples is just a convenience. In reality, first WebGPU parses the WGSL to make
-sure it's syntactically correct. Then, WebGPU looks at the `entryPoint`
-you specify. From there, it goes and looks at the parts that entryPoint references
-and nothing else for that entryPoint. It's useful because you don't have to type
-things like structures or binding and group locations twice if two or more shaders
-share bindings or structures or constants or functions. But, from the POV of WebGPU,
-it's as though you did duplicate all of them, once for each entryPoint.
+sure it's syntactically correct. Then, WebGPU looks at each `entryPoint`
+you specify, separately. It looks at the parts that each entryPoint references
+and nothing else. 
+
+Shared strings are useful because you don't have to type things like structures, 
+binding and group locations, constants, and functions twice if multiple shaders 
+share the resources/entities. But, from the POV of WebGPU, it's as though you did duplicate 
```

Comment:
> if multiple shaders share the resources/entities

I don't know what resources and entities are here. 

How about

> Shared strings are useful because multiple shaders can then share things like structures, binding and group locations, constants, and functions.

Reply:
much cleaner, i agree. 'resources/entities' being used nebulously + placeholder-y.

Reply:
[resolution commit](./144/commits/b7c35efa2a8bd63ce633ae1b5ad9b0ab0128e57e)